### ISSUE TITLE
VM service: Frame documentation is incorrect

### ISSUE THREAD

**Issue by nex3** [dart-lang/sdk#24144]
The [VM service documentation](https://github.com/dart-lang/sdk/blob/master/runtime/vm/service/service.md#frame) describes the `Frame` class as so:

```
class Frame extends Response {
  int index;
  @Function function;
  @Code code;
  @Script script;
  int tokenPos;
  BoundVariable[] vars;
}
```

However, in the actual JSON, the `script` and `tokenPos` properties are replaced by a single `location` property that contains a `SourceLocation`.

**Comment by johnmccutchan** [comment]
Thanks for catching this. @turnidge FYI.

**Comment by turnidge** [comment]
Fixed in bfd59ed7993dc8718b2beb501b429a69fc4c1973.
